lib/virt_autotest/utils.pm

```diff
@@ -64,6 +64,23 @@ sub is_pv_guest {
     return $guest =~ /\bpv\b/;
 }
 
+#Check if guest is SLE with optional filter for:
+#Version: <=12-sp3 =12-sp1 >11-sp1 >=15 15+ (>=15 and 15+ are equivalent)
+#usage: guest_is_sle($guest_name, '<=12-sp2')
+sub guest_is_sle {
+    my $guest_name = lc shift;
+    my $query = shift;
+
+    return 0 unless $guest_name =~ /sle/;
+    return 1 unless $query;
+
+    # Version check
+    $guest_name =~ /sles-*(\d{2})(?:-*sp(\d))?/;
```

Comment:
The guest naming rules are different in MU, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/virt_autotest/common.pm.

Reply:
Yes, I consider the MU guest names when I created the rule. I verified the function can determine 'sles12sp3PV', 'sles15PV', 'sles11sp4PVx32' and 'sles-15-SP4-64'. But more tests are still needed on MU.